update.sh

```diff
@@ -54,6 +54,21 @@ FROM scratch
 ADD $thisTar /
 EOF
 
+	if [ "$v" = 'xenial' ]; then
+		cat >> "$v/Dockerfile" <<'EOF'
+# delete all the apt list files since they're big and get stale quickly
+RUN rm -rf /var/lib/apt/lists/*
+# this forces "apt-get update" in dependent images, which is also good
+# (see also https://bugs.launchpad.net/cloud-images/+bug/1699913)
+EOF
+	else
+		cat >> "$v/Dockerfile" <<'EOF'
+# verify that the APT lists files do not exist
+RUN [ -z "$(ls -A /var/lib/apt/lists)" ]
```

Comment:
Could this maybe be [ "$(apt-get indextargets | wc -l)" = "0" ] instead? We might change livecd-rootfs to leave behind empty auxfiles and partial directories at some point. Of course we can leave this as you have it for now and change the check when we need it if you'd prefer...

Reply:
Ahh, `apt-get indextargets`, absolutely a great idea. :+1:

Reply:
Updated!